src/Whoops/Exception/Frame.php

```diff
@@ -206,8 +206,8 @@ public function getFileLines($start = 0, $length = null)
 
             // Get a subset of lines from $start to $end
             if ($length !== null) {
-                $start  = (int) $start;
-                $length = (int) $length;
+                $start  = $start;
```

Comment:
I guess dropping all the casts could be considered a BC break. only because of a phpdoc comment saying it should be a int does not necessarily mean everyone out their used it only with ints.

Reply:
Maybe we should add tests instead? Or extend the `phpDocs`?